std_msgs/Header header

string message
uint8 barId # i.e. color
geometry_msgs/Point[] endcaps

std_msgs/MultiArrayDimension[] covDim
float64[] covariances